src/common/services/LedgerService.ts

```diff
@@ -36,9 +36,9 @@ export default class LedgerService extends WalletService {
     return LedgerTransportService.getInstance()
       .enqueueRequest(
         // eslint-disable-next-line no-async-promise-executor
-        (transport: TransportWebUSB) => new Promise<boolean>(async (resolve) => {
+        (transport: TransportWebUSB) => new Promise<boolean>((resolve) => {
           try {
-            await this.checkApp(transport);
+            this.checkApp(transport);
```

Comment:
I think this won't work as expected. As `checkApp` is a promise, we should get rid of the Promise constructor instead of the async/await.